packages/perseus/src/util/svg.ts

```diff
@@ -30,7 +55,7 @@ export class PathBuilder {
     ): PathBuilder {
         this.path.push({
             action: "C",
-            coords: [control1X, control1Y, control2X, control2Y, endX, endY],
+            args: [control1X, control1Y, control2X, control2Y, endX, endY],
```

Comment:
Question: Is the change to `args` to better match the code you are bringing in or does that just better reflect this information?

Reply:
Previously, this was called `coords` because all the values were in fact coordinates. But the `A` (arc) command for SVG paths has some "flag" arguments too, which determine the direction of the arc. I changed the name to reflect that.